model/task/db.go

```diff
@@ -985,72 +985,75 @@ func FindByExecutionTasksAndMaxExecution(taskIds []*string, execution int) ([]Ta
 
 type BuildVariantTuple struct {
 	BuildVariant string `bson:"build_variant"`
```

Comment:
Does this even need JSON tags? If we aren't returning this to users or reading JSON into it then I'd suggest removing it altogether.

Reply:
The bson tags are necessary for pipeline stages - there are no JSON tags currently

Reply:
Is this still necessary with the bsonutil tags?

Reply:
Np! Yes we need them because the bsonutil tags resolve to the bson tags defined on the struct and `MustHaveTag` will err if they do not exist.